dvc/remote/gdrive.py

```diff
@@ -155,51 +160,89 @@ def _validate_config(self):
             or not self._service_account_p12_file_path
         ):
             raise DvcException(
-                "To use service account please specify {}, {} and "
-                "{} in DVC config. Learn more at "
-                "{}.".format(
-                    "gdrive_service_account_email",
-                    "gdrive_service_account_p12_file_path",
-                    "gdrive_service_account_user_email (optional)",
-                    format_link("https://man.dvc.org/remote/modify"),
+                "To use service account set 'gdrive_service_account_email',\n"
+                "'gdrive_service_account_p12_file_path', and optionally "
+                "'gdrive_service_account_user_email'\nin DVC config. "
+                "Learn more at {}".format(
```

Comment:
```suggestion
                "To use service account, set `gdrive_service_account_email`,\n"
                "`gdrive_service_account_p12_file_path`, and optionally "
                "`gdrive_service_account_user_email`\nin DVC config. "
                "Learn more at {}".format(
```

Reply:
But actually, not sure `\n`s are needed, I remember @efiop mentioning somewhere that they're not? (This would apply to all the other comments.)

Reply:
Yes, this is arguable decision. There is one place in this PR (where hint is particularly long and has 1. 2. ,etc) - in this case I would keep it for sure ... in this specific case - I would keep them as well, just makes it easier to read, but agree that it puts some additional burden to maintain that properly + breaks dev logs. I made this decision nonetheless - users come first, developers after that :)

Reply:
OK but I would still use back quotes in all these. Single quotes are used to signal dynamic output in this repo from what I remember. Meaning output that comes from the `.format()` clause, especially if it's a variable.